okhttp/src/jvmMain/kotlin/okhttp3/internal/http2/Http2Reader.kt

```diff
@@ -308,6 +311,14 @@ class Http2Reader(
     if (length != 4) throw IOException("TYPE_WINDOW_UPDATE length !=4: $length")
```

Comment:
It seems a shame to not log the faulty frame in these two exception cases. Can we assume a rule that we intend to log each and every frame that comes in or goes out?

Reply:
Fixed.